google-cloud-bigtable/lib/google/cloud/bigtable/instance_admin_client.rb

```diff
@@ -0,0 +1,424 @@
+# frozen_string_literal: true
+
+require "google/cloud/bigtable/admin/v2"
+require "google/bigtable/admin/v2/instance_pb"
+
+module Google
+  module Cloud
+    module Bigtable
+      # Protobuf aliases
```

Comment:
Are these aliases needed? I don't see them being used. If the point of this gem is to intentionally leak the underlying protobuf types, what is the reason `Google::Bigtable::Admin::V2::Instance` is aliased as `Google::Cloud::Bigtable::Instance`?

Reply:
Added aliases for easy access of protobuf classes. Some of the APIs has a long chain of nested classes. i.e `modify_column_families` API has `Google::Bigtable::Admin::V2::ModifyColumnFamiliesRequest::Modification` and `Modification` contains another long chain of `ColumnFamily`. 
So aliases gives better aceesiblilty rather then using direct protobuf classes or hash objects.

These aliases mostly use by the bigtable client users to call apis.

I will move aliases to `Google::Bigtable` name space.
`Google::Bigtable::Admin::V2::Instance` => `Google::Bigtable::Instance`.